src/codeu/chat/client/core/Context.java

```diff
@@ -33,6 +34,10 @@ public Context(ConnectionSource source) {
     this.controller = new Controller(source);
   }
 
+  public ServerInfo getInfo() {
+    return ((View) view).getInfo();
```

Comment:
The problem is that getInfo() is only implemented in the client view, not the server view. So if I add it to BasicView, I just get an error saying that View.java (server) is not abstract and does not implement method getInfo()